specs/eventing/broker.md

```diff
@@ -145,6 +145,18 @@ The subscriber MAY receive a confirmation that a reply event was accepted by the
 Broker. If the reply event was not accepted, the initial event SHOULD be
 redelivered to the subscriber.
 
+Subscribers SHOULD acknowledge delivery of an event with an HTTP 2xx response code.
+
+Subscribers MAY respond to event delivery with one of the following responses
+to provide additional contextual meaning.
+
+    * 200: Acknowlegement of receipt with an event in the response.
```

Comment:
Does this mean that if they respond with a 200 but an empty body it’s an error?

Reply:
Not if we leave it at `MAY`. :smile:

Reply:
I really want to split this discussion into two.... 1) which 2xx is right when, and 2) what the response in each case MAY look like.  But I fear we might not be able to IF we're going to mandate something like "200 MUST include a non-empty body".  So, let me ask this... are we trying to mandate such things? And am I correct that this requirement is on the Sink (meaning the user's ksvc) and not on a Knative component ( like a broker) ?